### ISSUE TITLE
Rule E3693 - Validation Error for MonitoringRoleArn in Aurora Cluster Creation

### ISSUE THREAD

**Issue by ced455** [aws-cloudformation/cfn-lint#3800]
### CloudFormation Lint Version

c1.18.2

### What operating system are you using?

alpine

### Describe the bug

The CloudFormation linter rule E3693 - Additional properties are not allowed: 'MonitoringRoleArn' when creating Aurora cluster is incorrectly flagging the MonitoringRoleArn property in an AWS::RDS::DBCluster configuration with aurora-postgresql. This validation error conflicts with the official AWS CloudFormation documentation, which specifies that MonitoringRoleArn is a valid property for enabling enhanced monitoring on an Aurora PostgreSQL cluster.

Maybe I am missing something but the cfn-lint documentation does not help me yet

### Expected behavior

According to [AWS CloudFormation documentation for AWS::RDS::DBCluster](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html), MonitoringRoleArn is required to enable enhanced monitoring for an Aurora PostgreSQL cluster. Removing MonitoringRoleArn leads to a runtime error requesting this property, making it essential for Aurora clusters that utilize monitoring.

Additional Information: This validation rule prevents us from enabling monitoring on our Aurora PostgreSQL clusters. We need MonitoringRoleArn in the template to meet AWS requirements for enhanced monitoring, as specified in the documentation.

### Reproduction template

``` YAML
AuroraPgsqlDBCluster:
  Type: AWS::RDS::DBCluster
  Properties:
    Port: 5432
    Engine: aurora-postgresql
    EngineVersion: 16.4
    [...]
    DBClusterParameterGroupName: !Ref PgsqlDBClusterParameterGroup
    MonitoringRoleArn: !GetAtt MonitoringIAMRole.Arn
    PerformanceInsightsEnabled: true
    PerformanceInsightsRetentionPeriod: 7
    MonitoringInterval: 60

  AuroraPgsqlDBMaster:
    Type: AWS::RDS::DBInstance
    Properties:
      DBInstanceClass:
        Ref: DBInstanceClass
      DBClusterIdentifier: !Ref AuroraPgsqlDBCluster
      Engine: aurora-postgresql
      MonitoringInterval: 60
      AutoMinorVersionUpgrade: true
      DBSubnetGroupName: !Ref AuroraDBSubnetGroup
      PubliclyAccessible: true
      DBParameterGroupName: !Ref PgsqlDBInstanceParameterGroup


  AuroraPgsqlDBReadReplica:
    Type: AWS::RDS::DBInstance
    Properties:
      DBInstanceIdentifier: !Join [ '-', [ !Ref EnvironmentName, !Ref ApplicationCode, 'pgsql', 'reader' ] ]  # Nom des instances de lecture
      DBInstanceClass:
        Ref: DBInstanceClass
      DBClusterIdentifier: !Ref AuroraPgsqlDBCluster
      Engine: aurora-postgresql
      MonitoringInterval: 60
      AutoMinorVersionUpgrade: true
      DBSubnetGroupName: !Ref AuroraDBSubnetGroup
      PubliclyAccessible: true
      DBParameterGroupName: !Ref PgsqlDBInstanceParameterGroup

```

**Comment by kddejong** [comment]
[This launch](https://aws.amazon.com/about-aws/whats-new/2024/09/amazon-rds-performance-insights-supports-aurora-cluster-level-configuration/) has resulted in the change.  I'll make the updates today.
